Add readme to examples (#120)
google/flax: examples/README.md

> # FLAX Examples

Reviewer: ```suggestion
# Flax Examples
```

> This folder contains a collection of examples of implementations of various architectures in machine learning.

Reviewer: ```suggestion
This folder contains a collection of examples of implementations of various architectures and training procedures.
```
